### ISSUE TITLE
Ignoring E712 causes a spurious E704 for certain source code containing type annotations

### ISSUE THREAD

**Issue by csdev** [PyCQA/pycodestyle#710]
Given the following source code, saved as "sample.py":
```python
class MyClass:
    default: str = 'some text'
```
Running pycodestyle (or flake8) with E712 ignored causes a false positive E704:
```
$ pycodestyle --ignore=E712 sample.py 
sample.py:2:5: E704 multiple statements on one line (def)
```

The behavior is correct if E712 is not ignored:
```
$ pycodestyle sample.py
$ echo $?
0
```

It seems that the "def" in "default" is being parsed like a function definition instead of an attribute declaration.

Here is some information about my environment:
```
$ python --version
Python 3.6.3
$ pip freeze
flake8==3.5.0
mccabe==0.6.1
pkg-resources==0.0.0
pycodestyle==2.3.1
pyflakes==1.6.0
```

**Comment by sigmavirus24** [comment]
I believe this is fixed in master. Can you double check that?

**Comment by csdev** [comment]
Yes, it works properly on master. Just waiting for a release now :)
